rental.rb

```diff
@@ -0,0 +1,17 @@
+class Rental
+  attr_accessor :date, :books, :person
+
+  def initialize(date, books, person)
+    @date = date
+    @books = []
+    @person = person
+
+    books.each { |book| add_book(book) }
+    person.add_rental(self)
```

Comment:
- [ ] The rental is adding itself to the @book achieving the 'belongs-to' association. `book.rentals << self`. Please 🙏🏽 correct this line of your code, we can also check this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) about relationships.
- [ ] The rental is adding itself to the @person the 'belongs-to' association. `person.rentals << self`. Please 🙏🏽 correct this line of your code, we can also check this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) about relationships.